### DISCUSSION TITLE
Why are links lost during translation?

### DISCUSSION THREAD

**Discussion by boromyr** [immersive-translate/immersive-translate#1393] [Help]
###  Summary

The links are removed and replaced with plain text after translation, making the extension unusable. Is it possible to keep the links in the translation?

**Comment by theowenyoung** [comment]
Because the translation technology we currently use is pure text translation, some tag information will be lost.

It is recommended to use bilingual mode so that you can click on the links in the original text

**Comment by boromyr** [comment]
And would it not be possible to add the functionality to keep the links?
